neuroscout_cli/commands/run.py

```diff
@@ -1,133 +1,53 @@
-        no_drop = self.options.pop('--no-datalad-drop', False)
+        retcode = super().run()
         
-        out_dir = self.main_dir
-        out_dir.mkdir(exist_ok=True)
-
-        if neurovault not in ['disable', 'group', 'all']:
-            raise ValueError("Invalid neurovault option.")
-
         # Need to retrieve this from fitlins output once it's available
-        estimator = None
-        retcode = 0
-        if not upload_only:
-            fitlins_args = [
-                str(self.preproc_dir.absolute()),
-                str(out_dir),
-                'dataset',
-                f'--model={model_path}',
-                '--ignore=/(.*desc-confounds_regressors.*)/',
-                f'--derivatives={str(self.bundle_dir.absolute())} {str(self.preproc_dir.absolute())}',
-                f'--smoothing={self.options["--smoothing"]}',
-                f'--estimator={self.options["--estimator"]}',
-                f'--n-cpus={self.options["--n-cpus"]}'
-            ]
-
-            verbose = self.options.pop('--verbose')
-            if verbose:
-                fitlins_args.append('-vvv')
-            work_dir = self.options.pop('--work-dir', None)
-            if work_dir:
-                work_dir = str(Path(work_dir).absolute() / self.bundle_id)
-                fitlins_args.append(f"--work-dir={work_dir}")
-                
-            if self.options["--drop-missing"]:
-                fitlins_args.append(f"--drop-missing")
-                
-            # Save options used in execution
-            json.dump(self.options, (out_dir / 'options.json').open('w'))
-                        
-            # Call fitlins as if CLI
-            retcode = run_fitlins(fitlins_args)
-
-            if retcode != 0:
-                logging.error(
-                    "\n"
-                    "-------------------------------------------------------\n"
-                    "Model execution failed! \n"
-                    f"neuroscout-cli version: {VERSION}\n"
-                    "Update neuroscout-cli or revise your model, "
-                    "and try again \n"
-                    "If you believe there is a bug, please report it:\n"
-                    "https://github.com/neuroscout/neuroscout-cli/issues\n"
-                    "-------------------------------------------------------\n"
-                    )
-                return retcode
-
-        if neurovault != 'disable':
-            model = json.load(open(model_path, 'r'))
-            n_subjects = len(model['Input']['Subject'])
+        fitlins_args = [
+            str(self.preproc_dir),
+            str(self.main_dir),
+            'dataset',
+            f'--model={self.model_path}',
+            '--ignore=/(.*desc-confounds_regressors.*)/',
+            f'--derivatives={str(self.bundle_dir)} {str(self.preproc_dir)}',
+        ]
+        
+        # Append pass through options
+        fitlins_args.append(list(self.options['fitlings_args']))
```

Comment:
should the `--model`, `--ignore`, and `--derivatives` arguments be removed from fitlins_args (if present) or would you want users to be able to overwrite those arguments?

minor typo
```suggestion
        fitlins_args.append(list(self.options['fitlins_args']))
```